src/ocamlorg_frontend/components/icons.eml

```diff
@@ -0,0 +1,21 @@
+(** Using icons from {:https://heroicons.com/} *)
+
+let downloads =
+  <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 pr-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Comment:
One problem with extracting the icons (and the main reason we haven't done that) is that they aren't reusable if you hardcode the size here.

It should be possible to make them w-full and h-full and encode the size on the parent element though. I'd try this approach if you want to extract the icons.

Reply:
I thought about this and decided that it'll be better when we have a second place we want to the icon.
I think we don't want too many different sizes across the website so we probably want the sizes defined in `icons.eml`. Not sure how to allow different sizes for now